redis/datastore.go

```
@@ -3,20 +3,37 @@ package redis
+	if err != nil {
+		return err
+	}
+
+	return nil
 }
 
-func (d DataStore) GetTicks(ctx context.Context, assetID string, from, to *int64, limit int) ([]data.Tick, error) {
-	//TODO implement me
-	panic("implement me")
+func (d DataStore) GetTicks(ctx context.Context, assetID string) (data.Tick, error) {
+	val, err := d.client.Get(ctx, assetID).Result()
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)